sites/partners/src/components/listings/PaperListingDetails/sections/DetailAdditionalDetails.tsx

```diff
@@ -8,11 +8,33 @@ import SectionWithGrid from "../../../shared/SectionWithGrid"
 const DetailAdditionalDetails = () => {
   const listing = useContext(ListingContext)
 
+  const getRequiredDocuments = () => {
+    let documentsExist = false
+    const documents = Object.keys(listing?.requiredDocumentsList ?? {}).map((document) => {
+      if (document) {
```

Comment:
issue: This detail section is showing all of the required documents options rather than just the ones selected.
<img width="432" height="316" alt="Image" src="https://github.com/user-attachments/assets/47fd76a0-f906-4b0d-a2a8-e979fb631d51" />

 That's because you are only looking at the "keys" instead of the values. I recommend changing to use "entries" instead. Something like:
```
  const documents = Object.entries(listing?.requiredDocumentsList ?? {}).map(
      ([document, value]) => {
        if (value) {
```